File: 프로그래머스/2/42584. 주식가격/주식가격.py
def solution(prices):
    answer = []
    # 전체 리스트 돌거임
    for i in range(len(prices)):
        sec = 0
        # i보다 큰 인덱스랑 이제 비교할거임 ,,
        for j in range(i + 1, len(prices)):
            if prices[i] <= prices[j]:
                sec += 1
            else:
                sec += 1
                break
        answer.append(sec)
    return answer

File: 프로그래머스/2/42584. 주식가격/주식가격.java
public class Solution {
    public int[] solution(int[] prices) {
        int[] answer = new int[prices.length];
        
        for (int i = 0; i < prices.length; i++) {
            int sec = 0;
            
            // i보다 큰 인덱스들을 비교
            for (int j = i + 1; j < prices.length; j++) {
                sec++;
                
                // 가격이 떨어지면 break
                if (prices[i] > prices[j]) {
                    break;
                }
            } 
            answer[i] = sec;
        }
        return answer;
    }
}
